recipes/arg_router/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, rmdir
+from conan.tools.cmake import CMake, cmake_layout
+
+
+class arg_routerRecipe(ConanFile):
+    name = "arg_router"
+    license = "BSL-1.0"
+    homepage = "https://github.com/cmannett85/arg_router"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ command line argument parsing and routing."
+    topics = ("cpp", "command-line", "argument-parser", "header-only")
+
+    # CMake >= 3.18 is required but the recipe for it is not compatible with Conan v2, so just use
+    # the latest at the time of writing
+    tool_requires = "cmake/3.25.3"
+    requires = "boost/1.74.0", "span-lite/0.10.3"
```

Comment:
Is there a reason to not use the latest versions?

That is our recommend practice https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#dependencies :)